phoenix-core/src/it/java/org/apache/phoenix/end2end/AlterTableWithViewsIT.java

```diff
@@ -1149,10 +1152,8 @@ public void testDroppingIndexedColDropsViewIndex() throws Exception {
             byte[] viewIndexPhysicalTable = viewIndex.getPhysicalName().getBytes();
             assertNotNull("Can't find view index", viewIndex);
             assertEquals("Unexpected number of indexes ", 2, view.getIndexes().size());
-            assertEquals("Unexpected index ",  fullNameViewIndex1 , view.getIndexes().get(0).getName()
```

Comment:
Why is this change needed?

Reply:
The asserts here assume a certain ordering of indexes in the view's PTable but that ordering does not hold if the view PTable was refreshed in the client's cache. So I changed the asserts to look for the index name in a list of expected index names.

Reply:
Need to look into this test

Reply:
We do not invalidate cache when we drop indexes on the server side as part of a drop column statement.

Reply:
Will make that design change as part of https://issues.apache.org/jira/browse/PHOENIX-7305 and revert this part of the test change.

Reply:
Reverted